### PR TITLE
Make nova compute deployable on SLE12

### PR DESCRIPTION
**Pull request by rhafer**
This is a continuation of https://github.com/crowbar/barclamp-nova/pull/400

It addresses the remaining feedback and fixes some issues found during testing.

### CODE REVIEW

File: chef/cookbooks/nova/recipes/compute.rb [hunk 1]
```diff
@@ -156,47 +187,27 @@ def set_boot_kernel_and_trigger_reboot(flavor='default')
...
-    bash "edit qemu config" do
-      only_if "cat /etc/libvirt/qemu.conf | grep 'user =' | grep -q -v '#{libvirt_user}' || cat /etc/libvirt/qemu.conf | grep 'group =' | grep -q -v '#{libvirt_group}'"
-      code <<-EOH
-       sed -i 's|user *=.*|user = "#{libvirt_user}"|g' /etc/libvirt/qemu.conf
-       sed -i 's|group *=.*|group = "#{libvirt_group}"|g' /etc/libvirt/qemu.conf
-      EOH
-      notifies :restart, "service[libvirtd]"
-    end
+  template "/etc/libvirt/qemu.conf" do
+    source "qemu.conf.sle12.erb" if node.platform == "suse" && node.platform_version.to_f >= 12.0
```
Comment (tboerger) [inline chef/cookbooks/nova/recipes/compute.rb]: Why are you doing it that way? You are familar with the ```template````

* ```templates/host-$fqdn/$source```
* ```templates/$platform-$platform_version/$source```
* ```templates/$platform/$source```
* ```templates/default/$source```
* ```templates/$source```

**Comment by tboerger** [comment]
Wrong used ```template```resource file finder -.-

File: chef/cookbooks/nova/recipes/compute.rb (hunk 1 above)
Comment (vuntz) [inline chef/cookbooks/nova/recipes/compute.rb]: @tboerger: see https://github.com/crowbar/barclamp-nova/pull/400#discussion_r22035556

Comment (tboerger) [inline chef/cookbooks/nova/recipes/compute.rb]: I do really dislike this solution -.-

Comment (rhafer) [inline chef/cookbooks/nova/recipes/compute.rb]: Me too. But I disliked the other solution even more. For the reasons outlined in https://github.com/crowbar/barclamp-nova/pull/400#discussion_r22035556

**Pull request merged**